gradio/themes/utils/theme_dropdown.py

```diff
@@ -7,7 +7,7 @@
 def create_theme_dropdown():
     import gradio as gr
 
-    asset_path = pathlib.Path(__file__).parent / "themes"
+    asset_path = pathlib.Path() / "themes"
```

Comment:
Yep this makes sense. Before this file was placed alongside `app.py` in the space but now it's imported from the lib so `__file__` was wrong.

Thanks @abidlabs !

Reply:
Thank you @freddyaboulton for the quick reply! Will merge then